app/views/reviews/index.html.haml

```diff
@@ -30,3 +30,4 @@
           %td.text-right
             = link_to "", edit_review_path(review), class: %i{fa fa-pencil}, title: "Edytuj recenzję"
             = destroy_button(review,"Czy na pewno chcesz usunąć tę recenzję?")
+		    = paginate @reviews
```

Comment:
Zrobiłam jak Pan napisał i jest: The line was indented 2 levels deeper than the previous line.